### PR TITLE
hotfix for #359 and #360: fallback for glyphs not in the postscript lookup table

### LINKED ISSUE #359: Changes are breaking something commit 1f4056d
Unfortunately I can't provide sample PDF this time.
Output is correct before commit 1f4056d.

Output with latest commit / release:
```
Notice: Undefined offset: 67 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 117 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 114 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 114 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 105 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 99 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 117 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 108 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 117 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 109 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 32 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 86 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 105 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 116 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 97 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 101 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 32 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 48 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 55 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 46 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 48 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091
Notice: Undefined offset: 53 in pdfparser\src\Smalot\PdfParser\Encoding\PostScriptGlyphs.php on line 1091

... more than 5000 similary errors here
```

text:

ż ś ą ą żą ń ą ą ą ą ą ą ą ą ę ś ę ń ą ą ą ń ę ż ą ś ś ż ą ą ż ą ą ą ż ą ę ą ń ż ń ą ą ś ą ą ą ę ż ęż ń ę ę ą ą ą Ś Ś ą ę ż żą ż ż ż ą ś ąś ń ń ń ń ą ę ś ść Ś ść ść ść ść ś ść ę ść ą ść ść ść ść ę ś ż ę ą ą ą

### LINKED ISSUE #360: Undefined offset: 97 in Smalot/PdfParser/Encoding/PostScriptGlyphs.php on line 1091
code :
```php
<?php
 
// Include Composer autoloader if not already done.
include 'vendor/autoload.php';
 
// Parse pdf file and build necessary objects.
$parser = new \Smalot\PdfParser\Parser();
$pdf    = $parser->parseFile('data.pdf');
 
// Retrieve all pages from the pdf file.
$pages  = $pdf->getPages();
 
// Loop over each page to extract text.
foreach ($pages as $page) {
    echo $page->getText();
}
```
i m try data.pdf, i try data in [demo](https://www.pdfparser.org/demo) success

errors :
```bash
riptGlyphs.php on line 1091
PHP Notice:  Undefined offset: 53 in /project/php-pdf/vendor/smalot/pdfparser/src/Smalot/PdfParser/Encoding/PostScriptGlyphs.php on line 1091
PHP Notice:  Undefined offset: 32 in /project/php-pdf/vendor/smalot/pdfparser/src/Smalot/PdfParser/Encoding/PostScriptGlyphs.php on line 1091
PHP Notice:  Undefined offset: 32 in /project/php-pdf/vendor/smalot/pdfparser/src/Smalot/PdfParser/Encoding/PostScriptGlyphs.php on line 1091
PHP Notice:  Undefined offset: 32 in /project/php-pdf/vendor/smalot/pdfparser/src/Smalot/PdfParser/Encoding/PostScriptGlyphs.php on line 1091
PHP Notice:  Undefined offset: 32 in /project/php-pdf/vendor/smalot/pdfparser/src/Smalot/PdfParser/Encoding/PostScriptGlyphs.php on line 1091
PHP Notice:  Undefined offset: 80 in /project/php-pdf/vendor/smalot/pdfparser/src/Smalot/PdfParser/Encoding/PostScriptGlyphs.php on line 1091
PHP Notice:  Undefined offset: 97 in /project/php-pdf/vendor/smalot/pdfparser/src/Smalot/PdfParser/Encoding/PostScriptGlyphs.php on line 1091
PHP Notice:  Undefined offset: 115 in /project/php-pdf/vendor/smalot/pdfparser/src/Smalot/PdfParser/Encoding/PostScriptGlyphs.php on line 1091
PHP Notice:  Undefined offset: 105 in /project/php-pdf/vendor/smalot/pdfparser/src/Smalot/PdfParser/Encoding/PostScriptGlyphs.php on line 1091
PHP Notice:  Undefined offset: 101 in /project/php-pdf/vendor/smalot/pdfparser/src/Smalot/PdfParser/Encoding/PostScriptGlyphs.php on line 1091
PHP Notice:  Undefined offset: 32 in /project/php-pdf/vendor/smalot/pdfparser/src/Smalot/PdfParser/Encoding/PostScriptGlyphs.php on line 1091
PHP Notice:  Undefined offset: 83 in /project/php-pdf/vendor/smalot/pdfparser/src/Smalot/PdfParser/Encoding/PostScriptGlyphs.php on line 1091

```

### PR DESCRIPTION
**Pull request by Connum**
This fixes the `Notice: Undefined offset` errors reported in #359 and #360 by implementing a check whether the postscript glyph name exists in the lookup table, otherwise returning the unaltered glyph. Also added a test case that would catch this issue.

Fixes #359
Fixes #360

### CODE REVIEW

File: tests/Integration/ParserTest.php [hunk 1]
```diff
@@ -210,6 +210,28 @@ public function testIssue334()
...
     }
+
+    /**
+     * Test that issue related pdf can now be parsed:
+     * Glyphs not in the Postscript lookup table would cause "Notice: Undefined offset"
+     *
+     * @todo Note that the "ł" in przepływu is decoded as a space character. This was already
+     * the case before the PR that caused this issue and is not currently covered by this
+     * test case. However, this issue should be addressed in the future and its fix
+     * can then be incorporated into this test by uncommenting the third assertion below.
```
Comment (k00ni) [inline tests/Integration/ParserTest.php:221]: Can you move that part to the commented out code in the test itself and put it in `/** */`, like

```php
/**
 * @todo note ...
 */
// $this->assertStringContainsString(...
```
This way its more clear why that code line is commented out and we don't remove it accidentally in a future release.

**Review by k00ni — CHANGES_REQUESTED** [review]
Thank you for providing a fix! Just have a small thing about the note.

File: tests/Integration/ParserTest.php (hunk 1 above)
Comment (Connum) [reply tests/Integration/ParserTest.php]: Absolutely! Done.

Comment (k00ni) [reply tests/Integration/ParserTest.php]: Thanks!

**Review by k00ni — APPROVED** [review]

**Comment by k00ni** [comment]
After the coding style problems is fixed I will merge it (and prepare another release).

**Comment by Connum** [comment]
> 
> 
> After the coding style problems is fixed I will merge it (and prepare another release).

The error message was quite dubious... After running php-cs-fixer locally, it changed `/**` to `/*` in the comment block - I hope that will do.

**Comment by k00ni** [comment]
My bad for pasting it with `/**`.

**Pull request merged**